tests/dialects/test_sqlite.py

```diff
@@ -140,6 +140,16 @@ def test_sqlite(self):
             read={"snowflake": "LEAST(x, y, z)"},
             write={"snowflake": "LEAST(x, y, z)"},
         )
+        self.validate_all(
+            "SELECT STR_TO_DATE('2022-01-01', '%Y-%m-%d')",
+            write={
+                "mysql": "SELECT STR_TO_DATE('2022-01-01', '%Y-%m-%d')",
+                "sqlite": "SELECT STRFTIME('%Y-%m-%d', '2022-01-01')",
+            },
+        )
+        self.validate_identity("STRFTIME('%Y-%m-%d %H:%M:%f', '2024-01-01 13:14:15.123')")
+        self.validate_identity("STRFTIME('%Y-%m-%d %I:%M:%S %p', '2024-01-01 01:14:15 PM')")
+        self.validate_identity("STRFTIME('%d-%b-%Y', '01-Jan-2024')")
```

Comment:
Let's try to increase the coverage here by testing how `STRFTIME` is transpiled to some other important dialects like postgres, mysql, duckdb

Reply:
Also, I'm curious about the last two tests here because SQLite seems to return `NULL` for them or something?

```sql
sqlite> SELECT STRFTIME('%Y-%m-%d %I:%M:%S %p', '2024-01-01 01:14:15 PM');

sqlite> SELECT STRFTIME('%d-%b-%Y', '01-Jan-2024');

```

Are these formats correct? Why are we not getting back a proper value?